### DISCUSSION TITLE
Policies when a user is part of team

### DISCUSSION THREAD

**Discussion by ajGulati05** [spatie/laravel-permission#2195] [Q&A]
Are policies automatically enabled to see if user is part of team to make changes? I do use the setPermissionId thats being called on each request. But do not think the polices work based on team. AKA I can make a change if belong to a different team, wondering if I did something incorrect or that is the case

Thank you

**Comment by parallels999** [comment]
Feel free to make a PR for any fix

**Comment by ajGulati05** [comment]
@parallels999  before I do that, I am curious if this is the expected behaviour or the issue is with my code?

**Reply by parallels999** [reply]
I am not using policies
